source/Preferences.cpp

```diff
@@ -52,6 +52,9 @@ namespace {
 	const vector<string> BOARDING_SETTINGS = {"proximity", "value", "mixed"};
 	int boardingIndex = 0;
 
+	const vector<string> PARALLAX_SETTINGS = {"off", "fancy", "fast"};
+	int parallaxIndex = 0;
```

Comment:
Should the default preference for parallax be "fast"?

Reply:
Ah k. Can you set this here to 1 and then delete the line in Load that sets the value to true?